src/commons/sideContent/content/SideContentDataVisualizer.tsx

```
@@ -193,10 +169,44 @@ export const SideContentDataVisualizer = connect(
+const DataVisualizerDefaultText: React.FC = () => {
+  const { t } = useTranslation('sideContent', { keyPrefix: 'dataVisualizer' });
+  return (
+    <p id="data-visualizer-default-text" className={Classes.RUNNING_TEXT}>
+      {t('defaultText')}
+      <br />
+      <br />
+      <Trans
+        ns="sideContent"
+        i18nKey="dataVisualizer.instructions"
+        components={[
+          <code>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)